src/components/picker/picker.tsx

```diff
@@ -140,15 +142,14 @@ export class Picker {
         this.handleInteract = this.handleInteract.bind(this);
         this.handleListChange = this.handleListChange.bind(this);
         this.handleStopEditAndBlur = this.handleStopEditAndBlur.bind(this);
+        this.handleListClosed = this.handleListClosed.bind(this);
+
+        this.portalId = createRandomString();
     }
 
     @Watch('value')
-    public onChangeValue(newValue, oldValue) {
+    public onChangeValue(newValue = [], oldValue = []) {
         this.chips = this.createChips(this.value);
-        if (!this.multiple) {
```

Comment:
Do we need to remove this or am I missing something? :slightly_smiling_face:  
If `multiple` is `false`, neither `newValue` or `oldValue` will be arrays and will not have the `.length` property

Reply:
the if statement made sure, that the chipset wouldn't set focus when selecting a value one a picker that only accepts one value. that makes sense, but with the dropdown list rendered in the body it gives you worse keyboard support, because the tab position will stay on limel-list and if you then try to tab to the next field you end up on the url field of your browser (in chrome at least). Do you have a better idea how we can change the tab position? I tried to set the tabindex on the chipset but that didn't work